src/platform/library/platform.c

```diff
@@ -61,3 +61,17 @@ int platform_init(struct sof *sof)
 
 	return 0;
 }
+
+#ifdef __ZEPHYR__
+/* Dummies for unsupported architectures */
```

Comment:
Should call this something else than dummy

Reply:
Can you use the word "stub" here instead of dummies.

Reply:
@lgirdwood done